src/config.js

```diff
@@ -6,33 +6,31 @@ const config = cfg({
   allowedOrigins: {
     env: 'ALLOWED_ORIGINS',
     type: 'string',
-    default: '*',
+    required: true,
   },
   allowedHeaders: {
     env: 'ALLOWED_HEADERS',
     type: 'string',
-    default: '*',
+    required: true,
   },
   appName: {
     env: 'APP_NAME',
     type: 'string',
-    default: 'fusion-node-express-api-skeleton',
+    required: true,
   },
   desiredPort: {
     env: 'PORT',
     type: 'integer',
-    default: '3000',
```

Comment:
This one should be fine to leave as defaulted to 3000. Heroku is the intended deploy platform and by default that's the port it will look for.